pennylane_lightning/core/src/simulators/lightning_kokkos/measurements/tests/Test_StateVectorKokkos_Measure.cpp

```diff
@@ -226,25 +226,30 @@ TEMPLATE_TEST_CASE("Probabilities", "[Measures]", float, double) {
         {{0, 1, 2},
          {0.67078706, 0.03062806, 0.0870997, 0.00397696, 0.17564072, 0.00801973,
           0.02280642, 0.00104134}},
-        {{0, 2, 1},
-         {0.67078706, 0.0870997, 0.03062806, 0.00397696, 0.17564072, 0.02280642,
-          0.00801973, 0.00104134}},
-        {{1, 0, 2},
-         {0.67078706, 0.03062806, 0.17564072, 0.00801973, 0.0870997, 0.00397696,
-          0.02280642, 0.00104134}},
-        {{1, 2, 0},
-         {0.67078706, 0.0870997, 0.17564072, 0.02280642, 0.03062806, 0.00397696,
-          0.00801973, 0.00104134}},
-        {{2, 0, 1},
-         {0.67078706, 0.17564072, 0.03062806, 0.00801973, 0.0870997, 0.02280642,
-          0.00397696, 0.00104134}},
-        {{2, 1, 0},
-         {0.67078706, 0.17564072, 0.0870997, 0.02280642, 0.03062806, 0.00801973,
-          0.00397696, 0.00104134}},
+        //{{0, 2, 1},
+        // {0.67078706, 0.0870997, 0.03062806, 0.00397696, 0.17564072,
+        // 0.02280642,
+        //  0.00801973, 0.00104134}},
+        //{{1, 0, 2},
+        // {0.67078706, 0.03062806, 0.17564072, 0.00801973, 0.0870997,
+        // 0.00397696,
+        //  0.02280642, 0.00104134}},
+        //{{1, 2, 0},
+        // {0.67078706, 0.0870997, 0.17564072, 0.02280642, 0.03062806,
+        // 0.00397696,
+        //  0.00801973, 0.00104134}},
+        //{{2, 0, 1},
+        // {0.67078706, 0.17564072, 0.03062806, 0.00801973, 0.0870997,
+        // 0.02280642,
+        //  0.00397696, 0.00104134}},
+        //{{2, 1, 0},
+        // {0.67078706, 0.17564072, 0.0870997, 0.02280642, 0.03062806,
+        // 0.00801973,
+        //  0.00397696, 0.00104134}}, // TODO: Fix LK out-of-order permutations
```

Comment:
Are we bringing these back?

Reply:
Not for this release, for now we raise an error. These are wrong, as with the LQ test results for the same ordering, so we will need to fix LK's transpose too later. TODO in there will be enough to track